prow/jobs/busola/busola-service-catalog-ui/busola-service-catalog-ui.yaml

```diff
@@ -0,0 +1,101 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/busola
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-kyma: "true"
+  run_if_changed: "^service-catalog-ui/|^components/|^shared/|^package.json"
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/bootstrap:PR-2951
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/busola/service-catalog-ui"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+presubmits: # runs on PRs
+  kyma-project/busola:
+    - name: pre-busola-service-catalog-ui
+      cluster: untrusted-workload
+      optional: false
+      <<: *job_template
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: master
+      branches:
+        - ^master$
```

Comment:
do we need `master` as well?
(ass the same comment to each place that has this change :) )